Makefile

```diff
@@ -54,6 +54,9 @@ init: node_modules ;
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
```

Comment:
Yeah, `default` IIRC builds everything. Linting is relatively cheap in terms of time, so I thought it would be better to run `lint` in the default target because _if_ the contributor introduced an error, `make` would fail much, much faster.

Maybe we should have `default` just build the English version though, and introduce an `all` target similar to the current `default`?